ddtrace/internal/tracemethods.py

```diff
@@ -1,19 +1,68 @@
-import typing  # noqa:F401
+from typing import List
+from typing import Tuple
 
 import wrapt
 
+from ddtrace.internal.utils.deprecations import DDTraceDeprecationWarning
+from ddtrace.vendor.debtcollector import deprecate
 
-def _parse_trace_methods(raw_dd_trace_methods):
-    # type: (str) -> typing.List[str]
-    """Return the methods to trace based on the specification of DD_TRACE_METHODS.
 
-    DD_TRACE_METHODS is specified to be FullyQualifiedClassOrModuleName[comma-separated-methods]
+def _parse_trace_methods(raw_dd_trace_methods: str) -> List[Tuple[str, str]]:
+    """Return a list of the module,methodname tuples to trace based on the
+    specification of DD_TRACE_METHODS.
+
+    DD_TRACE_METHODS is specified to be FullyQualifiedModuleName:comma-separated-methods;...
+    The square bracket notation will be deprecated in favor of this new syntax
 
     Note that support for wildcard methods ([*]) is not implemented.
```

Comment:
Is this not relevant to call out? Or are we just mentioning it in the public docs and not in this method comment?

Reply:
It's sort of out of place in the function that parses the new format, which doesn't allow any square brackets. If you want to keep it around, maybe move it to the legacy parser function?

Reply:
Oh I see what you're pointing out. In that case, I might just remove the square brackets from this comment since wild cards aren't supported with for this new notation either.